app/src/newdocs/content/core.mdx

```diff
@@ -1,11 +1,19 @@
 import ScrollableAnchor from 'react-scrollable-anchor'
+import { Link } from 'react-router-dom'
+
+import links from '$shared/../links'
 
 import docsStyles from '$newdocs/components/DocsLayout/docsLayout.pcss'
 
 <ScrollableAnchor id="intro-to-core"><div>
 
 ## Intro to Core
---content--
+
+<Link to={links.userpages.main}>Core</Link> is the collection of [open source](https://github.com/streamr-dev/streamr-platform), Streamr developed and supported applications that help you create, process, visualise and sell your realtime data. 
```

Comment:
Seems MDX is not applied if it begins with a tag:
<img width="792" alt="Screen Shot 2019-07-09 at 10 32 31" src="https://user-images.githubusercontent.com/1064982/60868103-f1308a00-a234-11e9-9fa9-e4a90a877516.png">

Reply:
Suggestion for copy: 
> Core is the collection of open source applications developed and supported by Streamr that help you create [...]

Reply:
Sounds much nicer 👍

Reply:
> Seems MDX is not applied if it begins with a tag:
> <img alt="Screen Shot 2019-07-09 at 10 32 31" width="792" src="https://user-images.githubusercontent.com/1064982/60868103-f1308a00-a234-11e9-9fa9-e4a90a877516.png">

Yep, noted in the PR description above - it's an MDX bug that happens when the first word is a JSX link. It's likely that the new update to the MDX library will fix this.

Reply:
Or add a filler word in front, like `The <Link ...>Core</Link>`, that should fix it.

Reply:
Fair enough, maybe Streamr Core then